src/main/java/com/hbm/blocks/generic/BlockAbsorber.java

```diff
@@ -1,39 +1,150 @@
 package com.hbm.blocks.generic;
 
+import java.util.List;
 import java.util.Random;
 
+import com.hbm.blocks.BlockEnumMulti;
+import com.hbm.blocks.IBlockMulti;
 import com.hbm.handler.radiation.ChunkRadiationManager;
+import com.hbm.lib.RefStrings;
 
-import net.minecraft.block.Block;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
 import net.minecraft.block.material.Material;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraft.util.StatCollector;
 import net.minecraft.world.World;
 
-public class BlockAbsorber extends Block {
+public class BlockAbsorber extends BlockEnumMulti implements IBlockMulti {
 
-	float absorb = 0;
+    // Enum for tiers they are in order of meta data, 0, 1, 2, 3 for Base, Red, Green, Pink
+    public static enum EnumAbsorberTier {
+        
+        BASE("Radiation Absorber", 2.5F, "absorber"),
+        RED("Enhanced Radiation Absorber", 10F, "absorber_red"),
+        GREEN("Advanced Radiation Absorber", 100F, "absorber_green"),
+        PINK("Elite Radiation Absorber", 10000F, "absorber_pink");
+        
+        public final String displayName;
```

Comment:
Don't put display names serverside, also these are seemingly not even used at all.